### PR TITLE
Standard library builitins

### PR DESCRIPTION
**Pull request by mariari**

### CODE REVIEW

File: stdlib/Prelude.ju [hunk 1]
```diff
@@ -0,0 +1,7 @@
+-- it seems custom arrow syntax is not operational
```
Comment (cwgoes) [inline stdlib/Prelude.ju:1]: hmm why not?

File: stdlib/Prelude.ju [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+let (->) = %Builtin.ArrowOmega
+
+declare infixr (->) 1
+
+declare infixr (:) 2
```
Comment (cwgoes) [inline stdlib/Prelude.ju:7]: is `(:)` a builtin?

Comment (mariari) [reply stdlib/Prelude.ju:7]: yes, not one you can define though.

**Review by cwgoes — APPROVED** [review]
ACK, you can merge this if you want

**Pull request merged**